test/controllers/hosts_controller_test.rb

```diff
@@ -31,6 +31,18 @@ def host_attributes(host)
     assert_template 'index'
   end
 
+  test "should get csv index with data" do
+    host = FactoryGirl.create(:host, :with_hostgroup, :with_environment, :on_compute_resource)
```

Comment:
I wanted to test the last report field as well but I think it failed due to the way the report factory works.